db/fixtures/miq_event_definitions.csv

```diff
@@ -70,6 +70,23 @@ request_host_shutdown,Host Shutdown Request,Default,host_operations
 request_host_reboot,Host Reboot Request,Default,host_operations
 request_host_enable_vmotion,Host Vmotion Enable Request,Default,host_operations
 request_host_disable_vmotion,Host Vmotion Disable Request,Default,host_operations
+
+#
+# Middleware operations
+#
+middleware_deployment_ok,EAP Application Deploy Succeeded,Default,middleware_operations
+middleware_deployment_error,EAP Application Deploy Failed,Default,middleware_operations
+middleware_deployment_remove_ok,EAP Application Undeploy Succeeded,Default,middleware_operations
+middleware_deployment_remove_error,EAP Application Undeploy Failed,Default,middleware_operations
+middleware_datasource_ok,EAP Datasource Create Succeeded,Default,middleware_operations
+middleware_datasource_error,EAP Datasource Create Failed,Default,middleware_operations
+middleware_datasource_remove_ok,EAP Datasource Delete Succeeded,Default,middleware_operations
+middleware_datasource_remove_error,EAP Datasource Delete Failed,Default,middleware_operations
+middleware_jdbc_ok,EAP JDBC Driver Deploy Succeeded,Default,middleware_operations
```

Comment:
I don't think a JDBC driver should be prefixed by EAP. JDBC is "just" JDBC. @jdoyleoss ?

Reply:
If we don't prefix it here, the JDBC events won't be displayed together with the rest (the list is ordered alphabetically by name). We can do that if we don't mind that.